pycalphad/core/phase_rec.pyx

```diff
@@ -47,56 +47,11 @@ cdef public class PhaseRecord(object)[type PhaseRecordType, object PhaseRecordOb
     @cython.boundscheck(False)
```

Comment:
(GitHub is not very nice about where it lets me put comments) Can you document the purpose of PhaseRecord (line 20). Mostly w.r.t abstracting the out the difference between Model and CompiledModel API. 

I'm actually not sure if things should be documented in the pyx or the pxd. In principle, someone who is going to use it should just have to look at the definition in the pxd and not worry too much about the implementation, but it looks like the .pyx are the ones that show up in the docs.